source/manual/rules-for-getting-production-access.html.md

```diff
@@ -11,17 +11,53 @@ These rules apply to developers, SREs, and technical architects in the GOV.UK pr
 
 ## What production access means
 
-- Permission to [read & write production and staging hieradata](https://github.com/alphagov/govuk-secrets/blob/main/puppet_aws/hieradata/production_credentials.yaml) in govuk-secrets using GPG
-- Permission to read & write from the Pass in govuk-secrets store using [GPG](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/.gpg-id)
-- Access to Production Deploy Jenkins and Staging Deploy Jenkins to deploy applications via the [GOV.UK Production GitHub team](https://github.com/orgs/alphagov/teams/gov-uk-production)
-- SSH access to production and staging servers via [govuk-puppet](https://github.com/alphagov/govuk-puppet)
+We have two types of production access:
+
+1. [Production Deploy access](#production-deploy-access)
+2. [Production Admin access](#production-admin-access)
+
+### Production Deploy access
+
+This level of access allows engineers to deploy code but not administer admin related
+systems. It should be granted to both civil servants and contractors as needed.
+
+Access includes:
+
+- Permission to [deploy apps](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#deployment) in Jenkins via the [GOV.UK Production Deploy Github team](https://github.com/orgs/alphagov/teams/gov-uk-production-deploy)
+- Permission to [merge pull requests](https://docs.publishing.service.gov.uk/manual/merge-pr.html#header) in continuously deployed applications
+- Readonly access to logging systems such as Logit, etc.
+- AWS readonly access via the `role_user_user_arns` role in [Staging](https://github.com/alphagov/govuk-aws-data/blob/main/data/infra-security/staging/common.tfvars) and [Production](https://github.com/alphagov/govuk-aws-data/blob/main/data/infra-security/production/common.tfvars)
+- "Normal" access to GOV.UK Signon in Staging and Production (with app permissions granted as needed)
```

Comment:
If I've understood correctly, Deployment vs Admin access means no difference in Signon?

```suggestion
- Full access to GOV.UK Signon in Staging and Production (with app permissions granted as needed)
```

Reply:
In the [spreadsheet](https://docs.google.com/spreadsheets/d/1oqy7tKpB8mHBhHQ9jAZu0NR0GKKZXOqtQGBKHYVnpmk/edit?pli=1#gid=0&range=E21:E22) it says it should be `Normal` for Production Deploy access

Reply:
Fair enough 👍 maybe be more specific about the category:

```suggestion
- "Normal" role in GOV.UK Signon on Staging and Production (with app permissions granted as needed)
```